Add .net50

nesc58 opened the pull request:
Try to solve issue https://github.com/gmarokov/dotnet-trx2sonar/issues/1

nesc58 committed:
Add .net50 ad target framework

nesc58 committed:
Try to update workflow to publish netcoreapp3.1 and .net50 application

nesc58 committed:
Try to build only with 5.0.x

nesc58 committed:
Try to solve build problems by updating actions

nesc58 committed:
Yaml fixed

gmarokov commented:
Hi @nesc58 thanks for the PR, the pipeline failed with missing target framework. I think instead of `net50` should be `net5.0`. Can you look at this? Cheers.

nesc58 committed:
Use documented framework name of .net 5.0

nesc58 commented:
@gmarokov  I have changed the framework to `net5.0`. Locally this works as well as the commits before.
I have changed the pipeline too because I think that the problem is there.
The pipeline before executes the commands side-by-side. But we need to install both versiosn: 3.1.x and 5.0.x

nesc58 commented:
When this works I would solve #3 next

nesc58 committed:
Remove matrix should fix github actions
